README.md

```diff
@@ -463,14 +464,14 @@ You can run the CCM locally on your laptop or VM, i.e. not in the cluster. This
 1. Build it for your local platform `make build`
 1. Set the environment variable `CCM_SECRET` to a file with the secret contents as a json, i.e. the content of the secret's `stringData`, e.g. `CCM_SECRET=ccm-secret.yaml`
 1. Set the environment variable `KUBECONFIG` to a kubeconfig file with sufficient access to the cluster, e.g. `KUBECONFIG=mykubeconfig`
-1. Set the environment variable `PACKET_FACILITY_NAME` to the correct facility where the cluster is running, e.g. `PACKET_FACILITY_NAME=ewr1`
+1. Set the environment variable `METAL_FACILITY_NAME` to the correct facility where the cluster is running, e.g. `METAL_FACILITY_NAME=ewr1`
 1. If you want to run the loadbalancer, and it is not yet deployed, run `kubectl apply -f deploy/loadbalancer.yaml`
-1. Enable the loadbalancer by setting the environment variable `PACKET_LB=metallb://`
-1. If you want to use a managed Elastic IP for the control plane, create one using the Equinix Metal API or Web UI, tag it uniquely, and set the environment variable `PACKET_EIP_TAG=<tag>`
+1. Enable the loadbalancer by setting the environment variable `METAL_LB=metallb://`
+1. If you want to use a managed Elastic IP for the control plane, create one using the Equinix Metal API or Web UI, tag it uniquely, and set the environment variable `METAL_EIP_TAG=<tag>`
 1. Run the command, e.g.:
 
 ```
-PACKET_FACILITY_NAME=${PACKET_FACILITY_NAME} PACKET_LB=metallb:// dist/bin/packet-cloud-controller-manager-darwin-amd64 --cloud-provider=packet --leader-elect=false --authentication-skip-lookup=true --provider-config=$CCM_SECRET --kubeconfig=$KUBECONFIG
+METAL_FACILITY_NAME=${METAL_FACILITY_NAME} METAL_LB=metallb:// dist/bin/cloud-provider-equinix-metal-darwin-amd64 --cloud-provider=metal --leader-elect=false --authentication-skip-lookup=true --provider-config=$CCM_SECRET --kubeconfig=$KUBECONFIG
```

Comment:
There `--cloud-provider=metal` doesn't agree with https://github.com/packethost/packet-ccm/pull/137/files#diff-d677710ef260f87ecc6eedb2d9bf090758eaa9aeb4fd917444ff83ade963ca60R21 and https://github.com/packethost/packet-ccm/pull/137/files#diff-216efb827497cdf8756c11ceadeee3f375a30034d4999a023ed351d2f68411fbR41 (`equinixmetal`)

Should we perhapsuse `equinix-metal` in these locations?

Reply:
I am not sure if you can use a `-` in the name. The only other one out there is `alibaba-cloud` and they set the provider name as `alicloud`. I am going to stick with `equinixmetal` for this.